.github/workflows/ci.yml

```diff
@@ -113,16 +113,79 @@ jobs:
             -   name: Checkout
                 uses: actions/checkout@v3
 
+            -   name: Setup PHP to install dependencies
```

Comment:
I believe you do not need to switch to PHP 5.5, if later you run composer-install with `composer config platform.php 2> /dev/null || composer config platform.php 5.5.0`

Reply:
Fair enough, but why the `composer config platform.php 2> /dev/null || ` ? AFAICS `composer config platform.php 5.5.0` should be sufficient.

Reply:
it was a fallback of mine if the file already had the platform.php configured ;) I'm fine to skip it